cfme/common/provider.py

```diff
@@ -3,6 +3,7 @@
 from manageiq_client.api import APIException
 
 import cfme
+import cfme.base.credentials
```

Comment:
OPTIONAL: @RonnyPfannschmidt please follow the way you used below - just import class as BaseCredential